formats/calendar/SRF_Calendar.php

```diff
@@ -357,7 +367,12 @@ function displayCalendar( $events ) {
 		}
 
 		$cur_month = self::intToMonth( $cur_month_num );
-		$cur_year = date( 'Y', time() );
+
+		if ( is_numeric( $this->mStartYear ) && ( intval( $this->mStartYear ) == $this->mStartYear ) ) {
```

Comment:
Since you removed the validation on behalf of  `time()` what happens when `$this->mStartYear` is  `-9000`? Does it still work without crashing due to out of bound limits?